packages/client-sdk-nodejs/src/batchutils/batch-props.ts

```diff
@@ -0,0 +1,36 @@
+import {CacheGet, CacheSet, CacheDelete} from '@gomomento/sdk-core';
+import {CacheClient} from '../cache-client';
+
+export const defaultMaxConcurrentRequests = 5;
+
+export interface BatchGetRequest {
+  cacheClient: CacheClient;
+  cacheName: string;
+  keys: Array<string>;
+  maxConcurrentGets?: number;
+}
+
+export type BatchGetResponse = Record<string, CacheGet.Response>;
+
+export interface BatchSetItem {
```

Comment:
I think you can get rid of this too, you want the API for `batchSet` to feel as similar as possible to the API for `cacheClient.set`. Yours will have the extra `cacheClient` argument at the beginning of its arg list, and the options arg for the maxConcurrentRequests, but otherwise you want the signatures to feel similar.

Which raises the question of whether we should have a TTL here; I think we probably should, you might want to fold that into your `options` argument?